students.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Student < Person
+  def initialize(_classroom, age, name = 'Unknown', parent_permission: true)
```

Comment:
- [ ] **Required:** In the initialize method param, you have preceded your classroom with an underscore "_". This is different from what you've used within the method body on line 6. Kindly unify them. If you want the attribute to be `_classroom`, then you should have: 
```
def initialize(_classroom, age, name = 'Unknown', parent_permission: true)
    super(age, name, parent_permission: parent_permission)
    @classroom = _classroom
 end
```

Or you could remove the underscore entirely from all.

Reply:
thank you they have been corrected as suggested